plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -276,16 +276,6 @@ protected CachingHiveMetastore(HiveMetastore delegate, OptionalLong expiresAfter
         partitionCache = partitionCacheFactory.buildCache(this::loadPartitionByName, this::loadPartitionsByNames);
     }
 
-    private static <K, V> LoadingCache<K, V> neverCache(com.google.common.base.Function<K, V> loader)
```

Comment:
Could you wait until https://github.com/trinodb/trino/pull/15838 lands? This PR conflict with the other one

Reply:
it was meant to help with https://github.com/trinodb/trino/pull/15838#discussion_r1086817324, but sure, can wait.